capture-sdk/sdk/src/main/java/net/gini/android/capture/noresults/NoResultsActivity.java

```diff
@@ -25,6 +25,10 @@
 import static net.gini.android.capture.camera.CameraActivity.RESULT_ENTER_MANUALLY;
 import static net.gini.android.capture.internal.util.ActivityHelper.interceptOnBackPressed;
 
+/**
+ * Internal use only
+ */
+
 /**
```

Comment:
[Request] Sorry, Lenci, for annoying you. Please merge these two javadoc comment blocks so that `Internal use only.` is shown in the generated reference documentation. Thank you!

P.S.: Don't forget the dot from the end of the sentence. 😉